Sources/PackageModel/ResolvedModels.swift

```diff
@@ -15,26 +15,56 @@ public final class ResolvedTarget: CustomStringConvertible, ObjectIdentifierProt
 
     /// Represents dependency of a resolved target.
     public enum Dependency: Hashable {
+        public static func == (lhs: ResolvedTarget.Dependency, rhs: ResolvedTarget.Dependency) -> Bool {
```

Comment:
Is it not possible to let compiler auto synthesize this?

Reply:
No, because we don't want to have equality of conditions.